### PR TITLE
[website] Add home page; use plotly for charts

### PR DESCRIPTION
**Pull request by jacek-oet**
See https://github.com/open-energy-transition/solver-benchmark/issues/6

### CODE REVIEW

File: website/compare.py [hunk 1]
```diff
@@ -23,6 +23,23 @@
...
     solver2_data = df[df["Solver"] == solver2]
     # Create the scatter plot
-    fig = create_comparison_chart(solver1_data, solver2_data, solver1, solver2)
+    run_time_fig = create_comparison_chart(
+        solver1_data,
+        solver2_data,
+        solver1,
+        solver2,
+        metric_name="Runtime (s)",
+        comparison_type="Runtime"
```
Comment (siddharth-krishna) [inline website/compare.py:32]:
```suggestion
        metric_name="runtime (s)",
        comparison_type="runtime"
```

File: website/compare.py [hunk 2]
```diff
@@ -23,6 +23,23 @@
...
+    )
 
-    st.pyplot(fig)
+    mem_use_fig = create_comparison_chart(
+        solver1_data,
+        solver2_data,
+        solver1,
+        solver2,
+        metric_name="Memory Usage (MB)",
+        comparison_type="Memory Usage"
```
Comment (siddharth-krishna) [inline website/compare.py:41]:
```suggestion
        metric_name="peak memory usage (MB)",
        comparison_type="peak memory usage"
```

Comment (siddharth-krishna) [reply website/compare.py:41]: Also please let's double check: is it MB or MiB? (I always forget the difference 😅 )

**Comment by jacek-oet** [comment]
The result of /usr/bin/time -v includes the line:

`Maximum resident set size (kbytes): x
`

However, the unit kbytes is not explicitly mentioned in the [Ubuntu Units Policy](https://wiki.ubuntu.com/UnitsPolicy). I believe this should be interpreted as kB because:

1 kB = 1,000 bytes (Note: small "k")
I found three sources that also mention this as referring to kilobytes.


> https://linux.die.net/man/2/getrusage
> https://man7.org/linux/man-pages/man2/getrusage.2.html
> *This is the maximum resident set size used (in kilobytes).

> *ru_maxrss    the maximum resident set size utilized (in kilobytes).
> https://manpages.ubuntu.com/manpages/bionic/man2/getrusage.2freebsd.html?_gl=1*kksf9u*_gcl_au*MTM2NzU0OTY2OC4xNzI0OTgxMzMw&_ga=2.180816673.782637710.1724978835-620433090.1724978835

**Review by siddharth-krishna — APPROVED** [review]
Thanks, this is great!

**Pull request merged**